app/views/admin/edition_images/_image_upload.html.erb

```diff
@@ -9,9 +9,9 @@
 ) do %>
   <%= render "govuk_publishing_components/components/file_upload", {
     name: "image[image_data][file]",
-    id: "edition_images_image_data_file",
+    id: "edition_images_file",
     hint: "Images can be JPEG, PNG, SVG or GIF files.",
-    error_items: @new_image.present? ? errors_for(@new_image.errors, :"image_data.file") : nil,
+    error_items: @new_image.present? ? errors_for(@new_image.image_data.errors, :"file") : nil,
```

Comment:
I'm struggling to understand why this changed, is it to change the format of the error messages?